src/components/Table/EditColumnsButton.test.tsx

```diff
@@ -0,0 +1,81 @@
+import { noop } from "src/utils";
+import { click, render } from "src/utils/rtl";
+import { EditColumnsButton } from "./EditColumnsButton";
+import { GridColumn } from "./GridTable";
+import { SimpleHeaderAndData } from "./simpleHelpers";
+
+type Data = { name: string | undefined; value: number | undefined };
+type Row = SimpleHeaderAndData<Data>;
+const nameColumn: GridColumn<Row> = { name: "name", header: "Name", data: ({ name }) => name };
+const valueColumn: GridColumn<Row> = { name: "value", header: "Value", canHide: true, data: ({ value }) => value };
+const actionColumn: GridColumn<Row> = {
+  name: "actions",
+  header: "Action",
+  canHide: true,
+  data: () => <div>Actions</div>,
+};
+
+describe("EditColumnsButton", () => {
+  it("Should render editColumnsButton", async () => {
```

Comment:
**suggestion:** Same request about changing "Should" to "should".  (also, I think it'd make sense to uppercase the "E" in "editColumnsButton")